.gitignore

```diff
@@ -28,3 +28,7 @@ benchmark/**
 
 # log file
 logs/
+
+*.jks
```

Comment:
why we need this gitignore?

Reply:
ignore test file.

Reply:
It's too wide, can you put it only in test directory? Or delete the mid file when testcase over?